### DISCUSSION TITLE
Share your GGUF conversions here !

### DISCUSSION THREAD

**Discussion by JorgeR81** [city96/ComfyUI-GGUF#144] [General]
I thought we could have a thread to share ( and maybe request :) GUFF conversions of image models.  

@city96 has list for base models on HF
https://huggingface.co/collections/city96/gguf-image-model-quants-67199ef97bf1d9ca49033234

---

But there are many great finetunes that don't have a conversion.

For instance, this model is built on the Flux Dev (de-distill), but has good results, with "just" 30 steps !
It's from the creator of the Realistic Vision (SD1.5) and RealVisXL models !
https://huggingface.co/SG161222/Verus_Vision_1.0b
https://civitai.com/models/883426/verus-vision-10b

I don't think this creator plans to make GGUF versions, because his other Flux models have been released for almost a month, and there are no GGUF versions yet.
https://civitai.com/models/788550/realflux-10b
https://huggingface.co/SG161222/RealFlux_1.0b_Schnell
https://huggingface.co/SG161222/RealFlux_1.0b_Dev

----

**EDIT:** 

Among many others, we now have conversions of these 2 finetunes:

- Verus Vision, thanks to user [LususNaturae](https://civitai.com/user/LususNaturae) ( see here: https://github.com/city96/ComfyUI-GGUF/discussions/144#discussioncomment-11579176 )

- Realflux-10b, thanks to user [Dar0z](https://civitai.com/user/Dar0z) ( see here: https://github.com/city96/ComfyUI-GGUF/discussions/144#discussioncomment-11691583 )

**Comment by JorgeR81** [comment]
**_PixelWave Dev 03_** - released by the creator in `Q4_K_M`, `Q_6_K` and `Q_8`
https://huggingface.co/mikeyandfriends/PixelWave_FLUX.1-dev_03
https://civitai.com/models/141592/pixelwave - _( some more info about the model in here )_

---

**_Acorn Is Spinning v1.1_**, `dev` and `Hyper 8 step` - converted by user to `Q4_K_M` and `Q5_K_M` ( from the FP8 original ) 
https://huggingface.co/sherlockbt/acorn-is-spinning-flux-guff/tree/main
https://civitai.com/models/673188?modelVersionId=978207 - _( these are also shared by the model's creator, with more info and images )_

---

**_Colossus Project 2.1 de-distilled_** - released by the the creator in `Q4_1` and `Q8_0`
https://civitai.com/models/833086?modelVersionId=997306
https://civitai.com/models/833086?modelVersionId=1003830

**Comment by city96** [comment]
***SD3.5-medium*** - converted by me to all available legacy and K quants above 3 bpw. Also has base FP16 gguf file.
https://huggingface.co/city96/stable-diffusion-3.5-medium-gguf
https://huggingface.co/stabilityai/stable-diffusion-3.5-medium
(_`SkipLayerGuidanceSD3` node seems to help a lot._)
(_`Q3_K_*` quants are already fairly low quality so I didn't do `Q2_K`_)

**Comment by JorgeR81** [comment]
**NepotismFUX V8** - released by the creator in `Q4_K_S` and `Q8_0` ( original in `FP16` ) 
https://civitai.com/models/618792?modelVersionId=1030306

----

**Jib Mix Flux V5** - released by the creator in `Q4_0`, ( original in `FP8` ) 
https://civitai.com/models/686814/jib-mix-flux?modelVersionId=1028151

----

**8 steps CreArt-Hyper-Flux-Dev V2.7** - released by the creator in `Q4_0`, ( original in `FP8` ) 
https://civitai.com/models/699688?modelVersionId=1029628

---

**Flux1-DedistilledMixTuned-V1** - released by the creator in `Q4_1` and `Q8_0`, ( original in `FP8` ) 
https://civitai.com/models/941929?modelVersionId=1054490

**Comment by YarvixPA** [comment]
**FLUX.1-Fill-dev-gguf** -  released by me
https://huggingface.co/YarvixPA/FLUX.1-Fill-dev-gguf

**Comment by JorgeR81** [comment]
**Artsy Dream v4** - released by the creator in `Q4_K_S` and `Q8_0` ( original in `FP16` )
https://civitai.com/models/870948?modelVersionId=1071073

---

**ultrarealFineTune** - released by the creator in `Q4_0` and `Q8_0` ( original in `FP16` )
https://civitai.com/models/978314/ultrareal-fine-tune

---

**Atomix Flux** - released by the creator in `Q4_0` and `Q8_0` ( original in `FP8` )
https://civitai.com/models/983139?modelVersionId=1101373

---

**Copax TimeLess Xplus 3** - released by the creator in `Q4_0` ( original in `FP8` )
https://civitai.com/models/118111?modelVersionId=1098346

---

**RedCraft | 红潮 CADS Commercial & Advertising Design System** - released by the creator in `Q4_0` ( original in `FP16` )
https://civitai.com/models/958009?modelVersionId=1079609

---

**MYHuman-墨幽随拍-Flux** - released by the creator in `Q4_0` ( original in `FP16` )

https://civitai.com/models/775057/myhuman-flux?modelVersionId=1080461

---

**2758 - Flux Asian Utopian** - released by the creator in `Q4_0`, `Q5_0` and `Q8_0` ( original in `FP16` )
https://civitai.com/models/705606?modelVersionId=1080655

**Comment by city96** [comment]
***Lightricks/LTX-Video*** (LTXV) - converted by me, has all available quants that made sense including FP32.
https://huggingface.co/city96/LTX-Video-gguf
https://huggingface.co/Lightricks/LTX-Video
(_Needs original VAE from [here](https://huggingface.co/Lightricks/LTX-Video/blob/main/vae/vae_diffusion_pytorch_model.safetensors)_)
(_`Q2_K` skipped for obvious reasons, see image below_)

![image](https://github.com/user-attachments/assets/3008e9d8-56c9-4283-9306-39f84e91c928)

**Comment by Kwisss** [comment]
this one would be interesting:
[https://huggingface.co/SG161222/RealVis_Medium_1.0b/tree/main](url)

**Reply by JorgeR81** [reply]
The link above seems to be broken. 
Here's an updated link: 

https://huggingface.co/SG161222/RealVis_Medium_1.0b/tree/main

**Comment by JorgeR81** [comment]
**STOIQO NewReality Alpha Two**, converted to `Q4_K` and `Q8_0` by user [LususNaturae](https://civitai.com/user/LususNaturae)
https://civitai.com/models/1029719?modelVersionId=1168583

**Verus Vision 1.0b**, converted to `Q8_0` by user [LususNaturae](https://civitai.com/user/LususNaturae)
https://civitai.com/models/1043778?modelVersionId=1171095

**Comment by city96** [comment]
**HunyuanVideo** - Converted by me from original bf16 `pt` file. Works with recently added native ComfyUI nodes.

https://huggingface.co/city96/HunyuanVideo-gguf

(*[VAE](https://huggingface.co/Kijai/HunyuanVideo_comfy/blob/main/hunyuan_video_vae_bf16.safetensors) / [FP16+FP8 Text Encoder](https://huggingface.co/Comfy-Org/HunyuanVideo_repackaged/tree/main/split_files/text_encoders)*)

![image](https://github.com/user-attachments/assets/50dc4cbc-f869-400d-b652-824358221884)

**Reply by city96** [reply]
Quantized version of the LLaMA model used as the text encoder: https://huggingface.co/city96/llava-llama-3-8b-v1_1-imat-gguf

**Reply by city96** [reply]
Also uploaded **FastHunyuan** by community request.

https://huggingface.co/city96/FastHunyuan-gguf

![image](https://github.com/user-attachments/assets/8f0e934b-0a54-4aaf-8eb6-7e4ee19c39a0)

**Comment by JorgeR81** [comment]
This user ( https://civitai.com/user/Dar0z ) made `Q4_K_S` conversions of some popular Flux finetunes:

- getphat FLUX Reality NSFW
- STOIQO Afrodite
- RealFlux 1.0b
- The Araminta Experiment
- AGFlux
- Vision Realistic
- XUER - One cyan ten color
- Flux_dev_fp8-Analog-art

**Comment by JorgeR81** [comment]
**Fluxmania III**, converted to `Q5_0` and `Q8_0`, by user [higegojira](https://civitai.com/user/higegojira)
https://civitai.com/models/1090700/gguf-fluxmania-iii

**Comment by hum-ma** [comment]
Quantized two pruned/light-weight Flux models:

https://huggingface.co/hum-ma/flux.1-lite-8B-GGUF
https://huggingface.co/hum-ma/Flex.1-alpha-GGUF

They even seem to work with Flux LoRas more or less, at least some that I tried. Other loras didn't give good results and with 4GB VRAM my Flux tests have been very limited.

**Comment by Njbx** [comment]
Someone convert the following models SkyReels T2V I2V ?
https://huggingface.co/Kijai/SkyReels-V1-Hunyuan_comfy/tree/main
